layout/templates/futureSuitability.mustache

```diff
@@ -11,8 +11,8 @@
 	  {{/futureSuitability-bullets-1}}</ul>
 	  <div class="figToggle showing" id="suitability"><p>Lake Count</p></div> 
 	  <div class="figToggle" id="figArea"><p>Total Lake Area</p></div>
-	  <object id="futureSuitabilityFig" type="image/svg+xml" class="svgFig" data="futureSuitability-desktop.svg"></object>
-	  <object id="futureSuitabilityFigArea" type="image/svg+xml" class="svgFig" data="futureSuitabilityFigArea-desktop.svg"></object>
+	  <object id="futureSuitabilityFig" type="image/svg+xml" class="svgFig" data="futureSuitability-desktop.svg" alt="Present and future count of lakes for each type of species dominance: walleye dominant, bass dominant, co-existance, or neither. Current distribution is 184 walleye lakes, 41 co-existance, 1236 bass, and 687 neither. Predicted distribution for 2040 to 2064 is 25 walleye lakes, 58 co-existance, 1857 bass, and 208 neither. Predicted distribution for 2065 to 2089 is 17 walleye lakes, 59 co-existance, 1961 bass, and 111 neither." title="Predicted distribution of dominant species for Wisconsin lakes by count"></object>
+	  <object id="futureSuitabilityFigArea" type="image/svg+xml" class="svgFig" data="futureSuitabilityFigArea-desktop.svg" alt="Present and future area of lakes for each type of species dominance: walleye dominant, bass dominant, co-existance, or neither. Current distribution is 326,183 acres walleye, 20,025 co-existance, 187,975 bass, and 217,207 neither. Predicted distribution for 2040 to 2064 is 47,813 acres walleye, 226,134 co-existance, 414,400 bass, and 63,043 neither. Predicted distribution for 2065 to 2089 is 41,342 acres walleye, 233,207 co-existance, 444,916 bass, and 31,923 neither.", title="Predicted distribution of dominant species for Wisconsin lakes by area"></object>
```

Comment:
These descriptions are crazy long, but I wasn't sure if we needed to include values to help people who can't see the fig understand what's going on here. Is there a different way we can have text say the counts/acres for each category?